website/docs/guides/setting-up-dex.md

```diff
@@ -60,7 +60,9 @@ kubectl create secret generic github-client \
 As we did before, we can use `HelmRepository` and `HelmRelease` objects to let
 Flux deploy everything.
 
-```yaml
+<details><summary>Expand to see resource manifests</summary>
+
+```yaml {43,52,66-74,88-89}
```

Comment:
what are your thoughts behind highlighting these lines?

Reply:
oh actually i meant to remove that highlighting